helm/kubernetes-node-exporter-chart/templates/daemonset.yaml

```diff
@@ -53,14 +57,16 @@ spec:
         - '--collector.sockstat'
         - '--collector.stat'
         - '--collector.systemd'
-        - '--no-collector.textfile'   # we don't use textfile collector.
         - '--collector.time'
         - '--collector.timex'
         - '--collector.uname'
         - '--collector.vmstat'
-        - '--no-collector.wifi'       # we don't use wifi.
         - '--collector.xfs'
-        - '--no-collector.zfs'        # we don't use zfs.
+
+        - '--no-collector.infiniband'
```

Comment:
Moving the disabled collectors to a separate group, as well as disabling infiniband (which we just don't use)